src/datasets/utils/resources/standard_licenses.tsv

```diff
@@ -19,7 +19,8 @@ Creative Commons Attribution Share Alike 4.0	cc-by-sa-4.0
 Creative Commons Attribution Non Commercial 3.0	cc-by-nc-3.0
 Creative Commons Attribution Non Commercial 4.0	cc-by-nc-4.0
 Creative Commons Attribution No Derivatives 4.0	cc-by-nd-4.0
-Creative Commons Attribution Non Commercial No Derivatives 4.0	cc-by-nc-nd-4.0
+Creative Commons Attribution Non Commercial No Derivatives  4.0 cc-by-nc-nd-4.0
+Creative Commons Attribution Non Commercial Share Alike 2.0 cc-by-nc-sa-2.0
```

Comment:
The problem persists: these 2 lines do not contain a tab "\t" separator (but just blank spaces):
```
 'Creative Commons Attribution Non Commercial No Derivatives  4.0 cc-by-nc-nd-4.0\n',
 'Creative Commons Attribution Non Commercial Share Alike 2.0 cc-by-nc-sa-2.0\n',
```

See other lines that are right:
```
 'Creative Commons Attribution Non Commercial Share Alike 3.0\tcc-by-nc-sa-3.0\n',
 'Creative Commons Attribution Non Commercial Share Alike 4.0\tcc-by-nc-sa-4.0\n',
 'Do What The F*ck You Want To Public License\twtfpl\n',
 'Educational Community License v2.0\tecl-2.0\n',
```